gulp/tasks/default.js

```diff
@@ -2,13 +2,13 @@ var gulp = require('gulp');
 var config = require('../config');
 
 gulp.task('watch', function() {
-    gulp.watch(config.browserify.watch,   ['browserify-app']);
+    gulp.watch(config.browserify.watch,   ['browserify']);
     gulp.watch(config.css.watch,          ['css']);
     gulp.watch(config.resources.watch,    ['resources']);
     gulp.watch(config.examples.watch,     ['examples']);
     gulp.watch(config.examplesJS.watch,   ['examplesJS']);
 });
 
-gulp.task('build-all', ['browserify-app', 'browserify-minify-app', 'vendor', 'css', 'resources', 'examples', 'examplesJS']);
+gulp.task('build-all', ['browserify', 'vendor', 'css', 'resources', 'examples', 'examplesJS']);
```

Comment:
Why was `browserify-minify-app` removed as a task? The task is still defined now in `browserify-task.js`, but is never run.

If we want to distinguish production builds from dev builds, we could do that, but note that it seems we are still building both css files in the `build-all` task.